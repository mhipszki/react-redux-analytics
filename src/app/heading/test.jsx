import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Heading from './';

test('styles an h1 element', () => {
  const component = shallow(<Heading />);
  expect(component).toMatchSnapshot();
});
