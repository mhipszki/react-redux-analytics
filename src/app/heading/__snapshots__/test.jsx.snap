// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles an h1 element 1`] = `
.c0 {
  font-size: 2rem;
  margin-top: 0;
}

<h1
  className="c0"
/>
`;
